### PR TITLE
CI exclude ObjectStore tests on normal runs

### PR DESCRIPTION
**Pull request by rullzer**
Since we can't run them anyway we better exclude them properly

I'm still thinking if we are better of doing this with groups? Or different suites.
Same approach I want to take with testing the different memcache backends. Isolate them in suites. And just run them with the required backend available.

CC: @MorrisJobke

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -72,3 +72,7 @@ Options -Indexes
 <IfModule pagespeed_module>
   ModPagespeed Off
 </IfModule>
+#### DO NOT CHANGE ANYTHING ABOVE THIS LINE ####
+
+ErrorDocument 403 /core/templates/403.php
+ErrorDocument 404 /core/templates/404.php
```
Comment (MorrisJobke) [inline .htaccess]: 👎

Comment (rullzer) [inline .htaccess]: fair enough..

**Comment by icewind1991** [comment]
:+1:

**Comment by rullzer** [comment]
@icewind1991 yeah I tend to agree after thinking more about it. Let me take a look tomorrow.

**Pull request closed without merging**